examples/textConnect/config_js/config.js

```diff
@@ -8,26 +8,29 @@ jQuery.noConflict();
 (function($, PLUGIN_ID) {
     'use strict';
 
-    //ユーザーの使用言語によって、表示する言語を変える。
     //To switch the language used for instructions based on the user's launguage setting
     var terms = {
         'en': {
             connectTitle: 'Fields to Connect',
             connectDescription: 'Please select the fields to connect. (5 fields max)',
-            errorMessage: 'Please select a field to display the combined texts.',
             delimiterTitle: 'Delimiter',
             delimiterDescription: 'Please specify the delimiter used between connected values. If not specified, values will be connected without delimiters.',
+            errorMessage: 'Please select a field to display the combined texts.',
```

Comment:
The error was slightly unclear what it was pointing to, when I checked the UI.
Let's call the setting name, so that the user knows what setting was missing.
Also, "display" implies that the data is shown, but not stored as a value.

Suggested error message as below:

```
"Fields to display the connected result" is a required setting.
Please select the output fields where the connected results will be inserted.
```

Reply:
I have changed the error message as mentioned above.